src/Uno.UI.Tests/Windows_UI_Xaml/Given_ThemeResource.cs

```diff
@@ -85,6 +87,8 @@ public void When_Inherited_In_Template_Applied()
 			control.InlineTemplateControl.ApplyTemplate();
 			control.TemplateFromResourceControl.ApplyTemplate();
 
+			control.ApplyTemplate();
```

Comment:
Necessary? `Test_Control` doesn't have a template, it's a `UserControl`.

Reply:
Yep it is. The control was failing without this.